.github/workflows/cron_tests_weekly.yml

```diff
@@ -111,7 +111,7 @@ jobs:
                                   python3-astropy \
                                   python3-scipy \
                                   python3-skimage \
-                                  python3-sklearn \                       
+                                  python3-sklearn \
```

Comment:
@larrybradley apparently these spaces were messing everything up.

Reply:
Ugh!  Those came from the GitHub suggestion.